ui/app/AppLayouts/Profile/popups/SendContactRequestModal.qml

```diff
@@ -40,6 +41,23 @@ StatusModal {
         property var lookupContact: Backpressure.debounce(root, 400, function (value) {
             root.contactsStore.resolveENS(value)
         })
+
+        function textChanged(text) {
```

Comment:
Mb `resolvePublicKey(chatKey)`  is better name for this method?

Reply:
This function does not resolve public key. It does a set of operations when text is changed (manually or pasted) , eg. sets some properties and also runs lookupContact which returns resolved name.